packages/dataviews/src/view-table.js

```diff
@@ -147,6 +137,32 @@ const HeaderMenu = forwardRef( function HeaderMenu(
 						) }
 					</DropdownMenuGroup>
 				) }
+				{ canAddFilter && (
+					<DropdownMenuGroup>
+						<DropdownMenuItem
+							prefix={ <Icon icon={ funnel } /> }
```

Comment:
Shouldn't we be consistent and use the `+` icon here as well? Funnel made sense when it was everywhere the same.

Reply:
-cc @jameskoster